roofit/roofitcore/inc/RooAbsFunc.h

```diff
@@ -30,26 +30,27 @@ class RooAbsFunc {
   inline RooAbsFunc(const RooAbsFunc& other) : _ncall(0), _dimension(other._dimension), _valid(kTRUE) { }
 
   inline virtual ~RooAbsFunc() { }
-  inline UInt_t getDimension() const { 
+  inline UInt_t getDimension() const {
     // Dimension of function
-    return _dimension; 
+    return _dimension;
   }
-  inline Bool_t isValid() const { 
+  inline Bool_t isValid() const {
     // Is function in valid state
-    return _valid; 
+    return _valid;
   }
 
   virtual Double_t operator()(const Double_t xvector[]) const = 0;
   virtual Double_t getMinLimit(UInt_t dimension) const = 0;
   virtual Double_t getMaxLimit(UInt_t dimension) const = 0;
 
-  Int_t numCall() const { 
-    // Return number of function calls since last reset
-    return _ncall ; 
+  /// Return number of function calls since last reset
```

Comment:
```suggestion
  /// Return number of function calls since last reset.
```
There should be always a period at the end of the sentence for doxygen, or is this not necessary anymore?